<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>watch_core</class>
 <widget class="qMRMLWidget" name="watch_core">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>187</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input volume:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="inputSelector">
       <property name="toolTip">
        <string>Pick the input to the algorithm.</string>
       </property>
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLScalarVolumeNode</string>
        </stringlist>
       </property>
       <property name="showChildNodeTypes">
        <bool>false</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
       <property name="SlicerParameterName" stdset="0">
        <string>inputVolume</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>watch_core</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>stopButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>watch_core</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>startButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>watch_core</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
